### DISCUSSION TITLE
SAS: Zombie Assault 3's Airbase

### DISCUSSION THREAD

**Discussion by Revnova** [nzp-team/nzportable#655] [Map Releases]
# [Download Here](https://github.com/nzp-team/nzportable/files/14269611/Airbase.zip)

This is based off a map from SAS: Zombie Assault 3. Airbase. Its just a quick map. so its not very detailed. 
its has:
1. the basic 4 perks
2. wall buys
3. 1 buyable door
4. the mystery box
5. Dogs
have fun 👍
# Screenshots
![fte-20240212160651-0](https://github.com/nzp-team/nzportable/assets/34180180/bd11a0ab-bcd5-4fe8-a5e7-7b9550d8749d)
![fte-20240212160707-0](https://github.com/nzp-team/nzportable/assets/34180180/025afa57-10d8-48b4-af99-ff96e2afe4b2)
![fte-20240212160747-0](https://github.com/nzp-team/nzportable/assets/34180180/708595a0-fb8c-4751-8d61-f8ce739d90ca)
![fte-20240212160805-0](https://github.com/nzp-team/nzportable/assets/34180180/fb8ade85-416e-4e74-a9d7-00c1be1f0552)

**Comment by MotoLegacy** [comment]
looks cool :)

**Comment by Revnova** [comment]
forgot to mention dogs. also fixed the dog spawns behind the door spawning cuz i forgot to name em.

**Comment by Conmiester** [comment]
I don't think I remember playing SAS: Zombie Assault 3 than the first two so long ago but, it looks just about as right from the original. Neat map regardless!

**Comment by Revnova** [comment]
forgot to include the sky. download updated

**Comment by V1cboi123** [comment]
Pretty good map. Didn't think anyone know about SAS 3 zombie Assault.
